manager/manager.go

```diff
@@ -110,10 +112,10 @@ type Manager interface {
 	CloseEventChannel(watch_id int)
 
 	// Get status information about docker.
-	DockerInfo() (DockerStatus, error)
+	DockerInfo() (docker.DockerStatus, error)
```

Comment:
Done. I moved it to `info/v1` for now, since that is our "internal" version at the moment. Once I get time to cleanup the versions properly, I'll add it to v2 as well.